test/helpers/envtest.go

```diff
@@ -114,6 +121,27 @@ func NewTestEnvironment() *TestEnvironment {
 	}
 }
 
+func initializeWebhookInEnvironment() {
+	namespacedScopeV1Beta1 := admissionv1beta1.NamespacedScope
```

Comment:
Tests are failing because those variables are declared but not used.
(In case you don't already know) you can use `make lint` or `make lint-full` (a little bit slower) to check these types of problems locally before pushing new changes